terraform/environments/production/dns.tf

```diff
@@ -90,6 +90,21 @@ resource "google_dns_record_set" "github-verification" {
   ]
 }
 
+# Microsoft 365
+
+resource "google_dns_record_set" "microsoft-verification" {
+  project      = module.google-cloud-project.project.project_id
+  managed_zone = module.google-cloud-dns.zone_name
+
+  name = module.google-cloud-dns.dns_name
```

Comment:
Isn't this going to conflict with the "google-mail" record below this?

Reply:
Hm, good catch. I'll need to add to the data there.